app/models/action.rb

```diff
@@ -1,6 +1,10 @@
 class Action < ActiveRecord::Base
   belongs_to :page
   belongs_to :action_user
+  after_create :update_page_action_count
 
+  def update_page_action_count
+    self.page.action_count = self.page.actions.count
```

Comment:
We do actually stop duplicate actions from happening, but from a higher level:

https://github.com/SumOfUs/Champaign/blob/development/app/services/manage_action.rb#L11

We could move that down to the model, but the best way to handle that would be with a `before_create` hook, or a `validates :unique` check.